gems/quilt_rails/CHANGELOG.md

```diff
@@ -5,7 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+
+### Changed
+
+- Refactor generators into smaller tasks.
+  Added the generation of a few more configuration files `.editorconfig`, `.eslintignore`, `.prettierignore`
```

Comment:
Should the `added` bit be under an `### Added` header? There's a lot going on here, maybe it's probably better to retitle this section to `### Added` instead of `### Changed`?

Reply:
sure thing